submissions/d-pozniak/html-css-popup/styles.css

```diff
@@ -0,0 +1,210 @@
+@font-face {
+    font-family: Antic;
+    src: url(../fonts/AnticSlab-Regular.ttf);
+}
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+html,
+body,
+div,
+span,
+a,
+img,
+footer,
+header,
+nav {
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+:focus {
+    outline: 0;
+}
+svg {
+    fill: #68838b;
+}
+
+/* HEADER */
+.header {
+    background: linear-gradient(180deg, rgba(191, 239, 255, 0.3), transparent);
+}
+.main-nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 120px;
+    max-width: 1560px;
+    margin: 0 auto;
+    padding: 0 10px;
+}
+.logo-wrapper {
+    display: flex;
+    align-items: center;
+    margin: 0 15px;
+}
+.logo-img {
+    fill: #68838b;
+}
+.logo-title {
+    font-size: 54px;
+    font-family: Antic, Arial, sans-serif;
```

Comment:
If you use this font for the whole page set this font for `html` or `body` elements.